src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -132,14 +111,17 @@ class ElectronicDeliveryForm extends React.Component {
           <h3>Chapter or Article Information</h3>
 
           <div className={`nypl-text-field ${errorClass.chapter}`}>
-            <label htmlFor="chapter" id="chapter-label">Chapter number or article title</label>
+            <label htmlFor="chapter" id="chapter-label">Chapter number or article title
```

Comment:
You forgot to update `htmlFor` to `chapter-title` since you updated the id for the input.